lib/taskana-core/src/main/resources/sql/db2/taskana_schema_update_3.0.0_to_4.0.0_db2.sql

```diff
@@ -13,3 +13,8 @@ CALL SYSPROC.ADMIN_CMD ( 'REORG TABLE HISTORY_EVENTS allow read access' );
 
 ALTER TABLE HISTORY_EVENTS ADD PRIMARY KEY (ID);
```

Comment:
Have you tried this on a real DB2 database?

Reply:
yes